CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.php

```diff
@@ -61,9 +68,9 @@ public function setDefaultValues() {
   public function buildQuickForm() {
     $priorFinancialPeriod = CRM_CloseAccountingPeriod_BAO_CloseAccountingPeriod::getPriorFinancialPeriod($this->_contactID);
     if ($priorFinancialPeriod) {
-      $this->assign('priorFinancialPeriod', $priorFinancialPeriod);
+      $this->assign('priorFinancialPeriod', $priorFinancialPeriod['M'] . '-' . $priorFinancialPeriod['Y']);
```

Comment:
Please don't hardcode formats like this. In this case, we should be using Date Format: Month and Year from civicrm/admin/setting/date?reset=1